tests/__init__.py

```diff
@@ -0,0 +1 @@
+from simulator.Pistons import Pistons
```

Comment:
Do you need this here?

Reply:
I do, remember I needed to put this last time for the test program to actually work.